resources/server/messages/messages.utils.ts

```diff
@@ -108,7 +109,7 @@ export async function createMessageGroupsFromPhoneNumber(
       sourcePhoneNumber,
     );
   } else {
-    await MessagesDB.createMessageGroup(sourcePhoneNumber, conversationId, sourcePhoneNumber);
+    await MessagesDB.createMessageGroup(tgtPhoneNumber, conversationId, sourcePhoneNumber);
```

Comment:
With this change, have you tested cases like, if you send a message to a person who does not have a conversation with you? Does it create the group properly?

Reply:
Yes it does